pkgs/applications/blockchains/monero-cli/default.nix

```diff
@@ -112,14 +109,24 @@ stdenv.mkDerivation rec {
       "-DCMAKE_CXX_FLAGS=-fpermissive"
     ];
 
-  outputs = [ "out" "source" ];
+  outputs = [
```

Comment:
This was on purpose. RFC 166 allows for short lists on a single line, but `nixfmt` is not smart enough to figure this out. https://github.com/NixOS/nixfmt/issues/206

Reply:
Ok. I don't mind changing it back, but why not go with what nixfmt does? Then you don't have to explain it to anyone else just following the contribution guidelines...

Reply:
No, it's fine. I'm just a bit salty at nixfmt making the code needlessly verbose.